apis/flowcollector/v1beta1/flowcollector_types.go

```diff
@@ -166,6 +167,66 @@ type EBPFMetrics struct {
 	Enable *bool `json:"enable,omitempty"`
 }
 
+// FlowFilterConfig defines the configuration of the flow filter.
+type FlowFilterConfig struct {
+	// FlowFilterIPCIDR defines the IP CIDR to filter flows by.
+	// Example: 10.10.10.0/24 or 100:100:100:100::/64
+	FlowFilterIPCIDR string `json:"cidr,omitempty"`
```

Comment:
nit: could you rename all the fields here without the `FlowFilter` prefix, so the go names reflect the json names?